docs/source/configuration/tracing.mdx

```diff
@@ -242,16 +241,20 @@ telemetry:
       # Optional collector configuration,
       collector:
         # Optional endpoint, either 'default' or a URL (Defaults to http://127.0.0.1:14268/api/traces)
-        endpoint: "http://my-jaeger-collector" 
+        endpoint: "http://${env.JAEGER_HOST}:14268/api/traces"
         username: "${env.JAEGER_USERNAME}"
         password: "${env.JAEGER_PASSWORD}"
 ```
 
-## OpenTelemetry Collector via OTLP
+Remember: If you're using Kubernetes, you can inject your secrets into configuration via environment variables.
```

Comment:
Do you mean 'secrets'? Or do you mean something like: "environment specific"?

Reply:
I'm going to remove this line as it could be applied to every exporter. We've documented env injection elsewhere.

Reply:
[bc6d6d2](https://github.com/apollographql/router/pull/3962/commits/bc6d6d2cc79904235b1eff6e15b5a99825c4845f)